crates/eips/src/eip4844/sidecar.rs

```diff
@@ -21,6 +21,7 @@ use alloc::vec::Vec;
 )]
 pub struct BlobTransactionSidecar {
     /// The blob data.
+    #[cfg_attr(feature = "serde", serde(deserialize_with = "deserialize_blobs"))]
```

Comment:
+ `debug_assertions`, this is not a problem with opt-level > 0

Reply:
cfg_attr(all(feature,debug_assertions)

Reply:
ah gotcha,